### ISSUE TITLE
[Suggestion]: Avoid using Git Sub-module 

### ISSUE THREAD

**Issue by licy183** [termux-user-repository/tur#10]
Currently, TUR uses `git sub-module` to get the `build.sh`s from `termux-main`, and uses a copy(fork) of the build system scripts of `termux-packages`. This makes it hard to keep up to date with the `termux-packages` repo.

Shall the master branch of `tur` repository only provide the `repo.json` and `tur` folders (and `README.md` etc.)? Actually it is enough for github action to perform a normal compilation. The action workflow can get the `master` branch of `termux-packages` and merge it with the `tur` repository before building packages. But this way will make it hard to build the packages in `tur` locally. Then a `bash script` could be used to get/update the master branch of `termux-packages`, merge it with the `tur` repository. Besides, add the files from `termux-packages` to `.gitignore`. Maybe this script will be named as `setup-environment.sh` or something else.

PS: This is just a suggestion and if it is inappropriate feel free to close it!

**Comment by kcubeterm** [comment]
packages: This directory is submodule which has termux-main packages build file, the reason to keep it here is, packages which we build may depends on, and dependency could be installed easily. So that submodule is only for reference. 

tur: This directory will contains what we build.

**Comment by licy183** [comment]
> packages: This directory is submodule which has termux-main packages build file, the reason to keep it here is, packages which we build may depends on, and dependency could be installed easily. So that submodule is only for reference.
> 
> tur: This directory will contains what we build.

Yes, but extra attention should be paid to make `packages` up to date with the `master` branch of `termux-packages`. If `termux/termux-packages` has changed, we should merge these changes to ``

**Comment by kcubeterm** [comment]
> Then a bash script could be used to get/update the master branch of termux-packages, merge it with the tur repository. Besides, add the files from termux-packages to .gitignore. Maybe this script will be named as setup-environment.sh or something else.

It's seems nice idea.
